src/Tribe/Attendee_Info/Template.php

```diff
@@ -0,0 +1,344 @@
+<?php
+/**
+ * Class Tribe__Tickets__Attendee_Info__Template
+ */
+class Tribe__Tickets__Attendee_Info__Template extends Tribe__Templates {
+
+	/*
+	 * List of themes which we may want to include fixes
+	 */
+	public $themes_with_compatibility_fixes = array(
+		'twentynineteen',
+		'twentyseventeen',
+		'twentysixteen',
+		'twentyfifteen',
+	);
+
+	/**
+	 * Initialize the template class
+	 */
+	public function init() {
+
+		// Spoof the context
+		add_filter( 'the_posts', array( $this, 'setup_context' ), -10 );
+
+		// Set and remove the required body classes
+		add_action( 'wp', array( $this, 'set_body_classes' ) );
+
+		// Choose the wordpress theme template to use
+		add_filter( 'template_include', array( $this, 'set_page_template' ) );
+
+		// Set the content of the page
+		add_action( 'loop_start', array( $this, 'set_page_content' ) );
+
+		// Modify the link for the edit post link
+		add_filter( 'edit_post_link', array( $this, 'set_edit_post_link' ) );
+
+		// Modify the page title
+		add_filter( 'document_title_parts', array( $this, 'modify_page_title' ), 1000 );
+	}
+
+	/**
+	 * Setup the context
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function setup_context( $posts ) {
+	    global $wp, $wp_query;
+
+	    // Bail if we're not on the attendee info page
+	    if ( empty( $wp->query_vars['attendee-info'] ) ) {
+			return $posts;
+		}
+
+		// Empty posts
+        $posts = null;
```

Comment:
Weird indentation on this method

Reply:
Thanks, changed my IDE recently. Will find a plugin to solve these ones.